tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.rst

```diff
@@ -705,7 +705,7 @@ First this function gets the axes for the trackpad and the joystick and assigns
 Then ``trackpad_vector`` and ``joystick_vector`` have their deadzones account for. The code for this is detailed in the article below, with slight changes as the code is converted from
 C# to GDScript.
 
-.. note:: You can find a great article explaining all about how to handle touchpad/joystick dead zones here: http://www.third-helix.com/2013/04/12/doing-thumbstick-dead-zones-right.html
+.. note:: You can find a great article explaining all about how to handle touchpad/joystick dead zones `here <https://web.archive.org/web/20191208161810/http://www.third-helix.com/2013/04/12/doing-thumbstick-dead-zones-right.html>`__.
```

Comment:
It's kind of redundant to have the same note twice, maybe we can remove one of them?

Reply:
I've removed the last note to avoid redundancy.